src/test/java/hudson/plugins/git/GitStatusCrumbExclusionTest.java

```diff
@@ -32,6 +34,19 @@
     private HttpServletResponse resp;
     private FilterChain chain;
 
+    private static String systemPropertyPreviousValue;
+
+    @BeforeClass
+    public static void setProps() {
+        systemPropertyPreviousValue = System.getProperty("hudson.security.csrf.CrumbFilter.UNPROCESSED_PATHINFO");
+        System.setProperty("hudson.security.csrf.CrumbFilter.UNPROCESSED_PATHINFO", "true");
+    }
+
+    @AfterClass
+    public static void unsetProps() {
+        System.setProperty("hudson.security.csrf.CrumbFilter.UNPROCESSED_PATHINFO", systemPropertyPreviousValue != null ? systemPropertyPreviousValue : "");
```

Comment:
Would prefer calls to `clearProperty` or `setProperty` depending on whether it was `null`.

Reply:
Done 👍